src/evo/deterministicmns.h

```diff
@@ -399,8 +399,15 @@ class CDeterministicMNList
         if (oldValue == newValue) {
             return;
         }
-        DeleteUniqueProperty(dmn, oldValue);
-        AddUniqueProperty(dmn, newValue);
+        static const T nullValue;
+
+        if (oldValue != nullValue) {
```

Comment:
Why not to delete existing null values here?

Reply:
Because there is an assertion in `DeleteUniqueProperty` that makes sure we delete something that actually exists.

Reply:
I see now,  thank you for clarification.)